recipes/bob.ap/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: bob.ap
+  version: {{ version }}
+
+source:
+  fn: bob.ap-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.ap/bob.ap-{{ version }}.zip
+  md5: 49f89a84c3c2cb20aa0ffa49d39415a7
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bob.extension
+    - bob.blitz
+    - bob.core
+    - bob.sp
+    - scipy
+    - cmake
+    - pkg-config
+    - gcc     # [linux]
```

Comment:
Ok, then it is fine. The main point here is we try not to use the `gcc` package unless it is strictly required. So, if we required newer C++ features or we need a Fortran compiler or similar.

We have several proposals that may eliminate this package on Linux in the future, but it is a pretty controversial and complex topic so it may be a while before we get there.